apps/web/src/pages/activities/components/ActivityStatistics.tsx

```diff
@@ -0,0 +1,80 @@
+import { useQuery } from 'react-query';
+import styled from '@emotion/styled';
+import { useMantineTheme } from '@mantine/core';
+import { getActivityStats } from '../../../api/activity';
+import { formatNumber } from '../../../utils';
+import { colors } from '../../../design-system';
+
+export function ActivityStatistics() {
+  const { data: activityStats, isLoading: loadingActivityStats } = useQuery<{
+    yearlySent: number;
+    monthlySent: number;
+    weeklySent: number;
+  }>('activityStats', getActivityStats);
+  const isDark = useMantineTheme().colorScheme === 'dark';
+
+  return (
+    <>
+      {!loadingActivityStats ? (
+        <ContentWrapper>
+          <StatisticsBox>
+            <StyledNumber isColored={false} data-test-id="activity-stats-weekly-sent">
+              {formatNumber(activityStats?.weeklySent ? activityStats?.weeklySent : 0, 0)}
+            </StyledNumber>
+            <StatsLabel isColored={false} isDark={isDark}>
+              This week
+            </StatsLabel>
+          </StatisticsBox>
+          <StatisticsBox>
+            <StyledNumber isColored>
+              {formatNumber(activityStats?.monthlySent ? activityStats?.monthlySent : 0, 0)}
+            </StyledNumber>
+            <StatsLabel isColored isDark={isDark}>
+              This month
+            </StatsLabel>
+          </StatisticsBox>
+          <StatisticsBox>
+            <StyledNumber isColored={false}>
+              {formatNumber(activityStats?.yearlySent ? activityStats?.yearlySent : 0, 0)}
+            </StyledNumber>
+            <StatsLabel isColored={false} isDark={isDark}>
+              This year
+            </StatsLabel>
+          </StatisticsBox>
+        </ContentWrapper>
+      ) : null}
```

Comment:
This will create a layout shift when loaded, becoming from nothing to a div with height. I Would at least suggest showing the entire structure and only filling the Styled numbers once the loaded, and showing empty string when it's loading. We later can create a skeleton there.

For extra points you can use mantine skeleton https://mantine.dev/core/skeleton/ to fill in the StyledNumber while it's loading :)